### PR TITLE
MFT: New json parameters added

### PR DESCRIPTION
**Pull request by tomas-herman**

### CODE REVIEW

File: MC/config/QC/json/qc-mft-cluster.json [hunk 1]
```diff
@@ -39,7 +39,11 @@
           "query": "randomcluster:MFT/COMPCLUSTERS/0;clustersrof:MFT/CLUSTERSROF/0;patterns:MFT/PATTERNS/0"
         },
         "taskParameters": {
-          "myOwnKey": "myOwnValue"
+          "maxClusterROFSize" : "5000",
+          "maxDuration" : "60000",
+          "timeBinSize" : "0.1",
+          "ROFLengthInBC" : "198",
+          "geomFileName" : "/home/epn/odc/files/o2sim_geometry-aligned.root"
```
Comment (knopers8) [inline MC/config/QC/json/qc-mft-cluster.json:46]: i am not sure if MC is ran with this file at this location

Comment (tomas-herman) [reply MC/config/QC/json/qc-mft-cluster.json]: I have removed the full path and left only the file name.

**Review by chiarazampolli — APPROVED** [review]

**Comment by chiarazampolli** [comment]
This unfortunately does not work with the FST:
```
[11072:qc-task-MFT-MFTClusterTask]: Error in <TFile::TFile>: file /sw/BUILD/b9cc6c375d599dcef17c1834875b84f66eabf036/O2-full-system-test/full-system-test-sim//home/epn/odc/files/o2sim_geometry-aligned.root_geometry-aligned.root does not exist
[11072:qc-task-MFT-MFTClusterTask]: [17:39:12][FATAL] Failed to open file /sw/BUILD/b9cc6c375d599dcef17c1834875b84f66eabf036/O2-full-system-test/full-system-test-sim//home/epn/odc/files/o2sim_geometry-aligned.root_geometry-aligned.root
[11072:qc-task-MFT-MFTClusterTask]: For later analysis we write a core dump to core_dump_11072
```
Maybe you can take a look at how the other detectors do?

**Comment by tomas-herman** [comment]
Looking both at the json files in the MC folder and in the qc-sync folder everyone is using the geometry file name without the full path. 

MC QC of ITS-TPC matching:
https://github.com/AliceO2Group/O2DPG/blob/ba7ee0e8ed5e58853b5f6d7890071ce183322e65/MC/config/QC/json/ITSTPCmatchedTracks_direct_MC.json#L55

Sync QC of ITS-TPC-TOF:
https://github.com/AliceO2Group/O2DPG/blob/7c76924652b419e777265733ab527381d4d3a9ed/DATA/production/qc-sync/itstpctof.json#L55

So I have removed the full path and in all cases it is just the file name now. Let's see if the test is ok now.

**Comment by knopers8** [comment]
it looks ok now

**Pull request merged**